articles/modules/ROOT/pages/protecting-against-cypher-injection.adoc

```diff
@@ -1,34 +1,34 @@
 = Protecting against Cypher injection
 :slug: protecting-against-cypher-injection
-:author: Andrew Bowman
-:neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4
+:author: Andrew Bowman, Gemma Lamont
+:neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4, 5.x
```

Comment:
Might be good to remove the old versions prior to 4.4 and 5.x? 

The reason I mention this is because the Cypher compiler went from a Scala compiler to a Java compiler during this time, and there were many small inconsistencies in the escaping algorithm (Michael Simmons can tell you all about it).